content/.gitignore

```diff
@@ -1 +1 @@
-storage*
+storage/*
```

Comment:
I'm curious, why you always change this??

Reply:
haha, good question... 
it was wrong and `content/src/storage` (code) was being ignored by linters but inner modifications were being tracked by git since it's already tracked

by default, content service is using `./storage` folder to store all content files, so every time I start it in dev mode it is creating this folder inside `content/`

this `.gitignore` was introduced when we removed `bazel` which run services in other context and `storage` folder lives in other place